features/step_definitions/features_steps.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+Given('a feature {string} of {provider}') do |feature_name, provider|
+  @service_plan = provider.service_plans.where(name: "Basic").last
+  feature = @service_plan.service.features.find_or_create_by(name: feature_name, scope: "ServicePlan")
+end
+
+Given('an enabled feature {string} of {provider}') do |feature_name, provider|
+  @service_plan = provider.service_plans.where(name: "Basic").last
+  feature = @service_plan.service.features.find_or_create_by(name: feature_name, scope: "ServicePlan")
+  @service_plan.features << feature
+  @service_plan.save!
+end
+
+Then('there {is} no feature named {string}') do |is, string|  
+  feature_name = @service_plan.service.features.last.name
+  assert_not_equal feature_name, string
+end
+
+Then('there {is} feature named {string}') do |is, string|
+  feature_name = @service_plan.service.features.last.name
+  assert_equal feature_name, string
+end
+
+And('I disable feature {string}') do |string|
+  element = find('.fa-check-circle')
```

Comment:
2. This selects the icon `<i>`, I guess clicking on the icon does nothing.
The actual action is triggered by clicking on `<a>` which is the parent of this icon.

Taking into account point 1., I'd say you should actually first find the `tr` element - as the row that contains the name of the feature (that you capture in `{string}`), and from there find the `<a>` that corresponds to the icon.